src/components/policy/policy_external/test/sql_pt_representation_test.cc

```diff
@@ -67,33 +67,22 @@ namespace test {
 namespace components {
 namespace policy_test {
 
-namespace {
-const int32_t kPolicyTablesNumber = 27;
-}
-
 class SQLPTRepresentationTest : public SQLPTRepresentation,
                                 public ::testing::Test {
- public:
-  static const bool in_memory_;
-
  protected:
-  static SQLPTRepresentation* reps;
-  static const std::string kDatabaseName;
-  static utils::dbms::SQLQuery* query_wrapper_;
+  static std::shared_ptr<SQLPTRepresentation> reps;
```

Comment:
@ZhdanovP why all these pointers are static?

Reply:
@AKalinich-Luxoft because they are using in a static class